nuxeo-coldstorage-web/test/nuxeo-move-contents-to-coldstorage-button-test.html

```diff
@@ -29,29 +31,225 @@
+          //   expect(isElementVisible(confirmationDialog.querySelector('#confirm'))).to.be.true;
+          // });
+
+          //   suite('REST Calls', () => {
+          //     let confirmationDialog, button;
+
+          //     setup(async () => {
+          //       button = fixture('docs_provided');
+          //       button.documents = documents;
+          //       Polymer.dom.flush(button);
+          //       const action = button.shadowRoot.querySelector('.action');
+
+          //       MockInteractions.tap(action);
+
+          //       confirmationDialog = button.shadowRoot.querySelector('#contentsToMoveDialog');
+          //       if (!isElementVisible(confirmationDialog)) {
+          //         await waitForEvent(confirmationDialog, 'iron-overlay-opened');
+          //       }
+          //     });
+
+          //     test('Should not do any REST call when cancel button is pressed', () => {
+          //       MockInteractions.tap(confirmationDialog.querySelector('#cancel'));
+
+          //       expect(server.getRequests().length).to.equal(0);
+          //     });
+
+          //     test('Should submit request when confirm button is pressed', () => {
+          //       server.respondWith('POST', '/api/v1/automation/Document.MoveToColdStorage');
+          //       MockInteractions.tap(confirmationDialog.querySelector('#confirm'));
+
+          //       const request = server.getLastRequest('POST', '/api/v1/automation/Document.MoveToColdStorage');
+          //       expect(request).to.exist;
+          //       expect(request.body).to.be.have.property('input', 'docs:123,456');
+          //     });
+
+          //     test('Should fire "notify" event when server returns deletion error', async () => {
+          //       server.rejectWith('POST', '/api/v1/automation/Document.MoveToColdStorage', { response: { status: 404 } });
+          //       MockInteractions.tap(confirmationDialog.querySelector('#confirm'));
+
+          //       const event = await waitForEvent(button, 'notify');
+          //       expect(event.detail).to.exist.and.to.have.key('message');
+          //       expect(event.detail.message).to.be.equal('moveDocumentsContentsToColdStorage.error');
+          //     });
+
+          //     test('Should fire "refresh" and "notify" events when server returns deletion error', async () => {
+          //       server.respondWith('POST', '/api/v1/automation/Document.MoveToColdStorage');
+          //       MockInteractions.tap(confirmationDialog.querySelector('#confirm'));
+
+          //       await Promise.all([waitForEvent(button, 'refresh'), waitForEvent(button, 'notify')]).then((events) => {
+          //         expect(events[1].detail).to.exist.and.to.have.key('message');
+          //         expect(events[1].detail.message).to.be.equal('moveDocumentsContentsToColdStorage.success');
+          //       });
+          //     });
+          //   });
+          // });
         });
-      });
+      });   
```

Comment:
all tests are down :( I thought only few of them. It seems that all tests are not passing.

Reply:
as we discussed, I removed the tests instead of commenting out and working on fixing some of them now. For the rest, we will address them as part of https://jira.nuxeo.com/browse/NXP-30322